### PR TITLE
Checkout: Thank you page is feature aware.

### PR DESCRIPTION
**Pull request by rodrigoi**

### CODE REVIEW

File: client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx [hunk 1]
```diff
@@ -12,26 +12,28 @@ import i18n from 'lib/mixins/i18n';
...
+			}
 
 			<PurchaseDetail
 				icon="stats-alt"
 				title={ i18n.translate( 'Stats from Google Analytics' ) }
 				description={ i18n.translate( 'Connect to Google Analytics for the perfect complement to WordPress.com stats.' ) }
 				buttonText={ i18n.translate( 'Connect Google Analytics' ) }
 				href={ '/settings/analytics/' + selectedSite.slug } />
+
+			{ plan.hasDomainCredit && <CustomDomainPurchaseDetail selectedSite={ selectedSite } /> }
```
Comment (rralian) [inline client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx]: You're changing the order of this. Previously CustomDomainPurchaseDetail was before PurchaseDetail. Shouldn't we retain the same order?

File: client/my-sites/upgrades/checkout-thank-you/premium-plan-details.jsx [hunk 2]
```diff
@@ -57,6 +59,7 @@ PremiumPlanDetails.propTypes = {
 		React.PropTypes.bool,
 		React.PropTypes.object
 	] ).isRequired,
+	selectedProduct: React.PropTypes.object,
```
Comment (rralian) [inline client/my-sites/upgrades/checkout-thank-you/premium-plan-details.jsx]: shouldn't this be `selectedFeature`?

File: client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx [hunk 3]
```diff
@@ -41,6 +43,7 @@ BusinessPlanDetails.propTypes = {
 		React.PropTypes.bool,
 		React.PropTypes.object
 	] ).isRequired,
+	selectedProduct: React.PropTypes.object,
```
Comment (rralian) [inline client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx]: shouldn't this be `selectedFeature`?

File: client/my-sites/upgrades/checkout-thank-you/index.jsx [hunk 4]
```diff
@@ -247,6 +249,10 @@ const CheckoutThankYou = React.createClass( {
 							purchases={ purchases }
 							registrarSupportUrl={ this.isGenericReceipt() ? null : primaryPurchase.registrarSupportUrl }
 							selectedSite={ selectedSite }
+							selectedFeature={ {
+								title: 'Google Analytics Integration',
+								plans: [ 'premium', 'business' ]
+							} } // findFeatureBySlugArtur( selectedFeature )
```
Comment (dmsnell) [inline client/my-sites/upgrades/checkout-thank-you/index.jsx]: Let's make sure we take this out before merging :wink:

File: client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx (hunk 1 above)
Comment (roundhill) [inline client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx]: We wanted to bump the context-relevant card to the top, which is why this is at the bottom now.

**Comment by roundhill** [comment]
Tested checkout Url from testing #4591, and it works fine :+1:

**Comment by rralian** [comment]
I'm not sure I understand what we're trying to do here. It looks like this just hides the "new themes" `<PurchaseDetail>` component when a `selectedFeature` is present. Is this just a necessary first step with other steps to follow? The lack of description left me confused here. /cc @rodrigoi

**Comment by roundhill** [comment]
Yeah the idea with this was to only show the Google Analytics page if any selectedFeature was present. We could make this more robust by actually doing a switch on the selectedFeature in the future, but for now I believe we are only passing through Google Analytics?

**Comment by rralian** [comment]
I don't know if that's happening or not. When I had tried testing this I wasn't sure what I was supposed to see or not see. But this was not launched to production and we still have an open issue referring to the feature here https://github.com/Automattic/wp-calypso/issues/4680. I had opened the issue because I saw that we had some work that was partially launched, but not with a good understanding of what this does.

What should we do here? Revert the PR? Launch to production?

**Comment by rralian** [comment]
I see, this _was_ launched to production, we have a feature flag but we're not checking it.

**Pull request merged**